face-kiosk/face-kiosk-application/src/main/java/com/pillar/skynet/facekiosk/application/Main.java

```diff
@@ -1,14 +1,29 @@
 package com.pillar.skynet.facekiosk.application;
 
+import com.pillar.skynet.facekiosk.detection.FaceDetector;
+import org.opencv.core.Core;
+import org.opencv.core.Rect;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static nu.pattern.OpenCV.loadShared;
+import static org.opencv.core.Core.NATIVE_LIBRARY_NAME;
+
 public class Main {
 
     private static final Logger LOG = LoggerFactory.getLogger(Main.class);
 
     public static void main(String[] s) {
         LOG.info("Eye see you...");
-    }
+		loadShared();
```

Comment:
Seems like we have a spacing issue.  Are the GAP tabs rules in play in your IDE?